graylog2-server/src/main/java/org/graylog2/configuration/validators/ElasticsearchVersionValidator.java

```diff
@@ -20,19 +20,21 @@
 import com.github.joschi.jadconfig.Validator;
 import com.google.common.collect.ImmutableList;
 import org.graylog2.plugin.Version;
+import org.graylog2.storage.versionprobe.SearchVersion;
 
 import java.util.List;
 
-public class ElasticsearchVersionValidator implements Validator<Version> {
+public class ElasticsearchVersionValidator implements Validator<SearchVersion> {
     private static final List<Version> SUPPORTED_ES_VERSIONS = ImmutableList.of(
+            Version.from(1, 0, 0),
             Version.from(6, 0, 0),
             Version.from(7, 0, 0)
     );
 
     @Override
-    public void validate(String name, Version value) throws ValidationException {
-        if (!SUPPORTED_ES_VERSIONS.contains(value)) {
-            throw new ValidationException("Invalid Elasticsearch version specified in " + name + ": " + value
+    public void validate(String name, SearchVersion value) throws ValidationException {
+        if (!SUPPORTED_ES_VERSIONS.contains(value.version())) {
```

Comment:
We could also make `SUPPORTED_ES_VERSIONS` a `List<SearchVersion>` in order to keep the product<->version relationships. Then, at this line we could write the condition as 
```
if (SUPPORTED_ES_VERSIONS.stream().noneMatch(supportedVersion -> supportedVersion.satisfies(value.distribution(), value.version()))) {
```